### PR TITLE
NSA-6289 Change new service added email add permission level

### PR DESCRIPTION
**Pull request by ronald-repka**

### CODE REVIEW

File: src/app/users/confirmNewUser.js [hunk 1]
```diff
@@ -146,17 +155,21 @@ const post = async (req, res) => {
...
+          req.session.user.lastName,
           org,
           serviceDetails.name,
-          roleDetails.map(i => i.name)
-        )
+          roleDetails.map((i) => i.name),
+          {
+            id: mngUserOrganisationDetails.role.id,
+            name: mngUserOrganisationDetails.role.name,
+          },
```
Comment (ipredoi) [inline src/app/users/confirmNewUser.js:167]: this object can be stored in a constant an used everywhere where notificationClient.sendServiceRequestApproved() is called

File: src/app/users/confirmNewUser.js [hunk 2]
```diff
@@ -187,13 +200,17 @@ const post = async (req, res) => {
...
+          req.session.user.email,
+          req.session.user.firstName,
+          req.session.user.lastName,
           org,
           serviceDetails.name,
-          roleDetails.map(i => i.name)
-        )
+          roleDetails.map((i) => i.name),
+          {
+            id: mngUserOrganisationDetails.role.id,
```
Comment (ipredoi) [inline src/app/users/confirmNewUser.js:210]: this object can be stored in a constant an used everywhere where notificationClient.sendServiceRequestApproved() is called

**Review by ipredoi — APPROVED** [review]

File: src/app/users/confirmNewUser.js (hunk 1 above)
Comment (ronald-repka) [reply src/app/users/confirmNewUser.js]: done

File: src/app/users/confirmNewUser.js (hunk 2 above)
Comment (ronald-repka) [reply src/app/users/confirmNewUser.js]: done

**Pull request merged**